### ISSUE TITLE
First person view, hide player model if zoomed-in

### ISSUE THREAD

**Issue by DaveScream** [ersh1/BG3_NativeCameraTweaks#19]
If vertical offset mult is set to ~1.0 and zoom in, it looks like first person BUT you see your character head. And when he moving all these body parts are visible in different positions.

Is it real to hide your player model when zoomed in ?

**Comment by darkstar123g28** [comment]
I agree, with the ExplorationZoomMin=0.001 parameter I get inside the model my character and see his eyes moving in front of me. The ability to move the camera forward relative to the character would solve the problem, or you could hide the character model as suggested above

**Comment by ersh1** [comment]
I might try to take a look at this when I work on the update for the new patch. Hiding the model definitely isn't gonna happen but moving the camera forward might be simple enough.

**Comment by darkstar123g28** [comment]
> I might try to take a look at this when I work on the update for the new patch. Hiding the model definitely isn't gonna happen but moving the camera forward might be simple enough.

thank you sir, this would make the game better, I would really like to replay the game with patch 8, with a first person view.

**Comment by darkstar123g28** [comment]
@ersh1 Thx for update for Patch 8, please tell me about the possibility of moving the camera forward/backward relative to the character for patch 8, I'm really looking forward to this opportunity.

**Comment by darkstar123g28** [comment]
@ersh1 

> I might try to take a look at this when I work on the update for the new patch. Hiding the model definitely isn't gonna happen but moving the camera forward might be simple enough.

Hello, please tell me, could you try to implement my request? I don't lose hope!)
